app/lib/analyzer/models.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:gcloud/db.dart' as db;
+import 'package:pub_semver/pub_semver.dart';
+
+import 'model_properties.dart';
+
+/// Analyzed package report.
+@db.Kind(name: 'PackageAnalysis', idType: db.IdType.String)
+class PackageAnalysis extends db.ExpandoModel {
+  String get packageName => id;
+
+  @db.ModelKeyProperty(required: true)
+  db.Key latestAnalysisKey;
```

Comment:
Why not make this `latestVersionAnalysisKey` ?

Alternatively we have to also remember what `analysisVersion` (aka package:pana version) this one has (similar to `PackageVersionAnalysis`), because when writing a new `Analysis` we need to know whether to update `PackageAnalysis` and/or `PackageVersionAnalysis` or not.